src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -211,14 +211,14 @@ opBuilderHelperIntTransformation(const std::string& targetField,
             catch (const std::exception& e)
             {
                 throw std::runtime_error(fmt::format(
-                    "\"{}\" function: Could not convert parameter \"{}\" to int",
+                    R"("{}" function: Could not convert parameter "{}" to int)",
                     name,
                     rightParameter.m_value));
             }
             if (IntOperator::DIV == op && 0 == std::get<int>(rValue))
             {
                 throw std::runtime_error(
-                    fmt::format("\"{}\" function: Division by zero", name));
+                    fmt::format(R"("{}" function: Division by zero)", name));
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings